packages/react-components/src/localization/LocalizationProvider.tsx

```diff
@@ -1,124 +1,47 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import React, { createContext, useState, useMemo, useContext, useCallback, useEffect } from 'react';
-import { defaultLocaleStringsLoader, locales } from './loadLocaleStrings';
-import defaultStrings from './translated/en-US.json';
-
-const LOCALE_CACHE_KEY = 'AzureCommunicationUI_Locale';
-
-export interface ILocale {
-  locale: string;
-  englishName: string;
-  displayName: string;
-  rtl: boolean;
-}
+import React, { createContext, useContext } from 'react';
+import en_US from './translated/en-US.json';
 
 /**
- * Collection of ILocale
+ * Locale keys to apply to component strings
  */
-export type ILocaleCollection = Record<string, ILocale>;
-
 export type ILocaleKeys = Record<string, string>;
 
-export interface ILocaleContext {
-  locale: ILocale;
-  locales: ILocaleCollection;
-  strings: ILocaleKeys;
-  setLocale: (locale: string, forceReload?: boolean) => void;
+/**
+ * Locale information to change strings in components
+ */
+export interface ILocale {
+  /** Set of strings for components */
+  localeStrings: ILocaleKeys;
```

Comment:
One of the recommendations from the meeting today was to avoid losing typing information in this interface.

So something like:

    export interface ILocate {
        messageThread: MessageThreadStrings,
        ...
    }

Reply:
Now I see. Thanks for the example. Done.